### PR TITLE
Setup Hello Microverse

### PR DESCRIPTION
**Pull request by AbenezerKb**
in this project, I :

set up a "Hello world" repository
Created a new branch called "my-second-branch", according to the GitHub flow
Added a .gitignore file.
Set up a linter for HTML and CSS
Added a descriptive README file to your project
Added 3 files (each one in a separate commit):
1. A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
2. A .css file that includes one class with styles for your header
3. A test.md file with the text "This file should be ignored by git"

### CODE REVIEW

**Review by muneebulrehman — COMMENTED** [review]
Hi @AbenezerKb , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no description for PR

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by AbenezerKb** [comment]
@muneebulrehman where should I add the description?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
```
Comment (HAMMAS-SALEEM) [inline README.md:54]: pr title (copy)


- [ ] Your branch name is not descriptive enough. You are using **add-homepage**, you can use  **feature/setup-hello-microverse** or **feature/initialize-hello-microverse** :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @AbenezerKb :wave:

### Highlights :1st_place_medal:
 
:heavy_check_mark: Following Linters
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Added .gitignore
:heavy_check_mark: Following the requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional Suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**